app/static/js/slcNewProposal.js

```diff
@@ -84,37 +84,9 @@ $(document).ready(function(e) {
         $("input[name=courseInstructorPhone]").focusout(blurHandler);
         $('#instructorTable').on('click', ".editButton", function() {
             var username=getRowUsername(this)
-            if ($(this).html() === 'Edit') {
-                $(this).html('Save')
-                $("#inputPhoneNumber-"+ username).focus()
-            } else {
-                // Save the phone number
-                var phoneInput = $("#inputPhoneNumber-" + username);
-                let isvalid = phoneInput.val().replace(/\D/g,"").length === 10;
-                let isempty = phoneInput.val().replace(/\D/g,"").length === 0;
-                if (!(isvalid || isempty)) {
-                    phoneInput.addClass("invalid");
-                    window.setTimeout(() => phoneInput.removeClass("invalid"), 1000);
-                    phoneInput.focus()
-                    return false;
-                }
-
-                $(this).html('Edit');
-                var instructorData = [username, phoneInput.val()]
-                $.ajax({
-                  url: "/updateInstructorPhone",
-                  data: JSON.stringify(instructorData),
-
-                  type: "POST",
-                  contentType: "application/json",
-                  success: function(response) {
-                      msgFlash("Instructor's phone number updated", "success")
-                  },
-                  error: function(request, status, error) {
-                    msgFlash("Error updating phone number", "danger")
-                  }
-                });
-            }
+            var phoneInput = "#inputPhoneNumber-" + username
```

Comment:
It's 2022 stop using var 

Either use const or let, or clock out and go back to gardening and watching The Price is Right.